testing/test_skipping.py

```diff
@@ -153,6 +154,71 @@ def test_func(self):
         assert skipped
         assert skipped.reason == "condition: config._hackxyz"
 
+    def test_skipif_additional_globals(self, testdir):
```

Comment:
Those tests are great, but please also add a test where different conftests provide different values, and the right `pytest_markeval_namespace` hook wins based on the test location. Something like:

```
tests/conftest.py   # pytest_markeval_namespace() -> {"arg": "root"}
tests/foo/conftest.py   # pytest_markeval_namespace() -> {"arg": "foo"}
tests/bar/conftest.py   # pytest_markeval_namespace() -> {"arg": "bar"}

tests/test_root.py  # single test with skipif("arg == 'root')
tests/foo/test_foo.py  # single test with skipif("arg == 'foo')
tests/bar/test_bar.py  # single test with skipif("arg == 'bar')
```

The above setup should make all tests skip. With the current code only `test_root` will be skipped.

Reply:
I'll add your suggested scenario.

Reply:
I'll add your suggested scenario.

Reply:
I'll add your suggested scenario.